torchvision/datasets/video_utils.py

```diff
@@ -2,20 +2,23 @@
 import math
 import warnings
 from fractions import Fraction
-from typing import List
+from typing import Any, Dict, List, Optional, Callable, Union, Tuple, TypeVar, cast
 
 import torch
 from torchvision.io import (
     _probe_video_from_file,
     _read_video_from_file,
     read_video,
     read_video_timestamps,
+    VideoMetaData,
```

Comment:
This seems to be unused as picked up by [`flake8`](https://app.circleci.com/pipelines/github/pytorch/vision/12677/workflows/919bab15-00d3-4f44-8b48-77dd7454a216/jobs/1005060).

Reply:
my bad!